server/eclipse-project/src/main/java/us/freeandfair/corla/model/Administrator.java

```diff
@@ -191,13 +183,27 @@ public void updateLastLoginTime() {
   }
   
   /**
+   * @return the last logout time.
+   */
+  public Instant lastLogoutTime() {
+    return my_last_logout_time;
+  }
+  
+  /**
+   * Updates the last logout time to the current time.
+   */
+  public void updateLastLogoutTime() {
+    my_last_logout_time = Instant.now();
+  }
+  
+  /**
    * @return a String representation of this contest.
    */
   @Override
   public String toString() {
     return "Administrator [username=" + my_username + ", type=" + 
            my_type + ", full_name=" + my_full_name + ", last_login_time = " + 
-           my_last_login_time + "]";
+           my_last_login_time + ", last_logout_time = " + my_last_logout_time + "]";
```

Comment:
By convention I have not been putting spaces in the `toString` around the equals signs (though apparently `last_login_time` was an exception to that rule).

Reply:
Got it; for the future.  That class is gone now, FWIW.

Reply:
Um... I doubt it. If you deleted the `Administrator` class, we'd have serious problems. :)